shared/model/transaction.ts

```diff
@@ -63,6 +68,7 @@ export interface IAssetSendStake extends IAsset {
 
 export interface IAssetVote extends IAsset {
     votes: Array<string>;
+    type?: VoteType;
```

Comment:
I think it is required field

Reply:
we reuse asset, in future yes it`s will be required